src/entity.js

```diff
@@ -115,6 +115,13 @@ for (let name in ABOUT.details.types) {
 			value: Object.freeze([]),
 			writable: true,
 		}
+		proto_desc.ReactedWith = {
+			value: Object.freeze({}),
+			writable: true,
+		}
+		// TODO: one thing to ask -- if i edit on a "replied" value, will LinkedUsers populate?
+		// why is ReactedWith not being set on messages in display_live?
+		// why is ReactedWith not set in the message cache?? is there a message cache?
```

Comment:
@12Me21 this acts weird -- i know there probably isn't a message cache anywwaaaay the